app/routes.py

```diff
@@ -1,2 +1,412 @@
+        blank_video_id = {"video_id": "can't be blank"}
+        input_errors.append(blank_video_id)    
+
+    customer_id = request_body.get("customer_id")
+    video_id = request_body.get("video_id")
+
+    if input_errors:
+        return jsonify(errors=input_errors), 400
+
+    video_thats_rented = Video.query.get(video_id)
+    if video_thats_rented.available_inventory == 0:
+        return jsonify(errors=(f"{video_thats_rented.title} is out of stock")), 400
+    else:
+        # increase the customer's videos_checked_out_count by one
+        customer_who_rents_video = Customer.query.get(customer_id)
+        customer_who_rents_video.videos_checked_out_count += 1
+        # decrease the video's available_inventory by one 
+        video_thats_rented = Video.query.get(video_id)
+        video_thats_rented.available_inventory -= 1
+
+        new_rental = Rental(
+            customer_id=request_body.get("customer_id"),
+            video_id=request_body.get("video_id"),
+            check_out=datetime.now(),
+            due_date=datetime.now() + timedelta(days=7)
+        )     # create a due_date that is 7 days from the current date
+    
+        db.session.add(new_rental)
+        db.session.commit()
+
+        response = {
+            'customer_id': new_rental.customer_id,
+            'video_id': new_rental.video_id,
+            'due_date': new_rental.due_date,
+            'videos_checked_out_count': customer_who_rents_video.videos_checked_out_count,
+            'available_inventory': video_thats_rented.available_inventory
+            }
+
+        return jsonify(response), 200
+    # have a success response that includes keys: 
+    # ['customer_id', 'video_id', 'due_date', 'videos_checked_out_count', 'available_inventory']);
+
+######################## POST RENTAL /rentals/check-in CRUD - CREATE ###################
+@rentals_bp.route("/check-in", methods=["POST"])
+def check_in_from_customer():
+    '''
+    when the customer checks-in/returns a rental/video, decrease the customer's videos_checked_out_count by one,
+    increase the video's available_inventory by one
+    '''
+    request_body = request.get_json() 
+
+    input_errors = []
+    if 'customer_id' not in request_body.keys(): 
+        blank_customer_id = {"customer_id": "can't be blank"}
+        input_errors.append(blank_customer_id)
+    if input_errors:
+        return jsonify(errors=input_errors), 400
+
+    else: # need to access all the data in customer_id --> not just the integer
+        customer_id = request_body.get("customer_id")
+        update_customer = Customer.query.get(customer_id)
+        customers_rental_ids = update_customer.get("rentals") # accesses the rentals column in the data customer table # get this row with the rental_id
+        for rental in customers_rental_ids:
+            if rental.id == update_customer:
+                down_one_count = update_customer.get("videos_checked_out_count") 
+                down_one_count -= 1
```

Comment:
update_customer is an instance of class Customer, while rental.id is will be a number, so this test will always fail (same logic in line 316).  As a result, the variables down_one_count and up_one_count on lines 323 & 324 will never be instantiated.